src/go/k8s/pkg/console/deployment.go

```diff
@@ -400,6 +411,17 @@ func (d *Deployment) getVolumes(ss map[string]string) []corev1.Volume {
 		})
 	}
 
+	if ss := d.consoleobj.Spec.SecretStore; ss != nil && ss.GCPSecretManager != nil && ss.GCPSecretManager.CredentialsSecretRef != nil {
```

Comment:
definitely not a fan of this style.

Reply:
Me too, but currently I would follow the convention. The next version of operator with helm would definitely have different style guide.